### ISSUE TITLE
no-curly-component-invocation: False positive for block invocation with inverse block

### ISSUE THREAD

**Issue by Turbo87** [ember-template-lint/ember-template-lint#1003]
Example:

```hbs
{{#some-component foo="bar"}}foo{{else}}bar{{/some-component}}
```

Component invocations with inverse blocks can't be converted to angle bracket components because inverse blocks with angle bracket components are not a thing that is supported or even exists at all.

**Comment by rwjblue** [comment]
Yep, agree we should also not flag these.
